### ISSUE TITLE
Data not available in benchmark-tutorial.ipynb

### ISSUE THREAD

**Issue by robmarkcole** [microsoft/PlanetaryComputerExamples#210]
Working through `benchmark-tutorial.ipynb` on a hub instance, the notebook states the data should be available in a volume but this is not the case:

```
---------------------------------------------------------------------------
AssertionError                            Traceback (most recent call last)
Input In [4], in <cell line: 5>()
      2 TRAIN_FEATURES = DATA_DIR / "train_features"
      3 TRAIN_LABELS = DATA_DIR / "train_labels"
----> 5 assert TRAIN_FEATURES.exists()

AssertionError: 
```

**Comment by TomAugspurger** [comment]
Thanks for the report. I was cleaning some things up in preparation for a Hub migration and completely forgot that this notebook existed :/

I'll need to think a bit about how to adjust for that. The notebook will fail to run in the meantime, unless you signed up for the competition: https://www.drivendata.org/competitions/83/cloud-cover/data/

**Comment by robmarkcole** [comment]
No problem! I did not sign up for the competition but appears as it has finished I cannot. 
I tried downloading but as this is 53GB I guess I wont have space. I only want to train on the NIR band, is there a way to request just that band?

**Comment by TomAugspurger** [comment]
> I tried downloading but as this is 53GB I guess I wont have space.

You might have space outside of your home directory (e.g. `/tmp`) but that's reset each time the notebook server restarts (https://planetarycomputer.microsoft.com/docs/overview/environment/#understanding-the-file-system).

> is there a way to request just that band?

I'm not sure. With the assets hosted by the [Planetary Computer](https://planetarycomputer.microsoft.com/catalog) you can access single bands. I don't recall how these assets were distributed, but it might have been a large gz or ZIP file.

**Comment by robmarkcole** [comment]
yes they are large gz. NP I will download elsewhere and create a NIR version for use here, thanks!
